platforms/FireTV/RefApp/app/src/main/java/com/truex/ctvwebrefapp/MainActivity.kt

```diff
@@ -68,6 +72,29 @@ class MainActivity : Activity() {
         webSettings.cacheMode = WebSettings.LOAD_NO_CACHE
     }
 
+    @JavascriptInterface
+    fun getAdvertisingId() {
+        getFireTVAdvertisingId()
+    }
+
+    private fun getFireTVAdvertisingId() {
+        try {
+            val cr = contentResolver
+            val advertisingID = Secure.getString(cr, "advertising_id")
```

Comment:
You have to also look to see if ad limiting is in effect.

I think it would be better to use the `AdvertisingIdClient` api class, and invoke these methods:
```
final Info adInfo = AdvertisingIdClient.getAdvertisingIdInfo(getApplicationContext());
adInfo.getId()
adInfo.isLimitAdTrackingEnabled()